deepchecks/utils/features.py

```diff
@@ -219,13 +220,22 @@ def _calc_importance(
         raise errors.DeepchecksTimeoutError('Permutation importance calculation was not projected to finish in'
                                             f' {permutation_importance_timeout} seconds.')
 
+    if hasattr(model, 'score'):
+        if hasattr(model, 'fit'):
+            scoring = None
```

Comment:
can you explain why do you check for fit?

Reply:
agree, we don't use fit anywhere